microsoft-edge/toc.yml

```diff
@@ -32,6 +32,9 @@
 
```

Comment:
change topmost What's New node from:
        - name: What's new
to:
        - name: What's New

Reply:
fixed